helm/default-apps-openstack/templates/cloud-provider-openstack-user-values.yaml

```
@@ -0,0 +1,14 @@
+apiVersion: v1
+data:
+  values: |
+    openstack-cinder-csi:
+      clusterID: {{ .Values.managementCluster }}_{{ .Values.clusterName }}
+    openstack-cloud-controller-manager:
+      controllerExtraArgs: |-
+        - --cluster-name={{ .Values.managementCluster }}_{{ .Values.clusterName }}
+kind: ConfigMap
+metadata:
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: {{ .Values.clusterName }}-cloud-provider-openstack-user-values
+  namespace: {{ .Release.Namespace }}
```

```suggestion
  namespace: {{ .Release.Namespace }}

```
All files should end with a newline. There are plugins for most editors to ensure this.